examples/model/erc20/src/lib.rs

```diff
@@ -29,6 +29,10 @@ use ink_model::{
     Contract,
     ContractDecl,
 };
+use ink_types_node_runtime::NodeRuntimeTypes;
```

Comment:
Have you checked if also the model and core examples are still working? I guess core example might no longer work, but that is not because of your PR ...

Reply:
Well, they compile and the tests run - didn't test them on chain...

Reply:
Actually turns out this was no longer working, fixed in latest commits